src/Makefile

```diff
@@ -36,6 +36,9 @@ NODEPS:=clean distclean
 
 # Default settings
 STD=-pedantic -DSERVER_STATIC=''
+ifeq ($(SERVER_TEST),yes)
+	STD +=-DSERVER_TEST=1
+endif
```

Comment:
It's good to have this make variable, but I think it's not right to add it to `STD`. We can add it to `FINAL_CFLAGS` instead, around 100 lines down .